File: ejercicios c++ echos por mi/ejercicios c++ pero en pyton/ejercicio_2.py
print("\033[2J\033[1;1f")
print("Para iniciar con el proceso de inscripcion de jugadores escriba si")
resp=str(input())
while (resp == "si"):
    cantidad=int(input("Cuantos jugadores desea ingresar?\n"))
    sumaedad=0
    sumaestatura = 0
    sumapeso = 0
    sumaimc=0
    for i in range (cantidad):

        nombre=str(input("Escriba el nombre del jugador\n"))

        edad=int(input("ingrese la edad del jugador\n"))
        while (edad <16 or edad > 18):    
            print ("La edad solo puede estar entre 16 y 18 años.")
            edad=int(input("ingrese la edad del jugador\n"))

        estatura=float(input("ingrese la estatura del jugador\n"))
        while (estatura < 1.55):
            print ("La estatura minima para estar en el equipo es de 1.55m.")
            estatura=float(input("ingrese la estatura del jugador\n"))

        peso=float(input("ingrese el peso del jugador\n"))
        while (peso < 50):
            print ("El peso no puede ser menor de 50kg.")
            peso=float(input("ingrese el peso del jugador\n"))

        imc=peso/(estatura*estatura)
        if (imc < 18.5):
            print(nombre," Tiene un peso demaciado bajo, su IMC es ",imc)
        elif (imc >=18.5 or imc <=24.9 ):
            print (nombre," Tiene un peso saludable, su IMC es ",imc)
        elif (imc >= 25.0 or imc <=29.0):
            print(nombre," Tiene sobrepeso, su IMC es ",imc)
        elif (imc > 30.0):
            print (nombre," Tiene sobrepeso, su IMC es ",imc)


        sumaedad = sumaedad + edad
        sumaestatura = sumaestatura + estatura
        sumapeso = sumapeso + peso
        sumaimc += imc

    promedad=sumaedad / cantidad
    promestatura=sumaestatura / cantidad
    prompeso=sumapeso / cantidad
    promimc=sumaimc / cantidad 

    print("El promedio de las edades es de ",promedad)
    print("El promedio de las estaturas es de",promestatura)
    print("El promedio de los pesos es de",prompeso)
    print("El promedio de los IMC es de",promimc)

    resp=str(input("¿Quiere repetir el proceso?\n"))

File: c++/ejercicio 2/ejercicio 2.cpp
#include <iostream>
using namespace std;
int main() {
  string nombre;
  string equipo;
  int edad;
  float estatura;
  float masa;
  int nregistro;
  float sumapeso;
  float sumaestatura;
  float sumaedad;
  int cont = 0;
  string resp;
  float promedad;
  float promestatura;
  float prompeso;
  int contjugadores;
  int opc ;

  do
  {
    promedad = 0;
    promestatura = 0;
    prompeso = 0;
    contjugadores = 0;
    do
    {
      cout << "Digite el nombre del jugador  " << endl;
      cin >> nombre;
      do
      {
        cout << "Recuerda que el rango de edades es entre 16 y 18 años" << endl;
        cout << "Digite la edad del jugador  " << endl;
        cin >> edad;
      } while (edad < 16 || edad > 18);

      do {
        cout << "Recuerde que la estatura debe ser mayor a 1.55 m" << endl;
        cout << "Digite la estatura del jugador " << endl;
        cin >> estatura;
      } while (edad < 1.55);

      do {
        cout << "Recuerde que el peso no puede ser menor a 50 Kg" << endl;
        cout << "Digite el peso del jugador   " << endl;
        cin >> masa;

      } while (masa < 50);

      float imc = masa / (estatura * estatura);
      if (imc < 18.5) {
        cout << nombre << " esta bajo de peso y su imc es de " << imc << endl;
      } else if (imc >= 18.5 || imc <= 24.9) {
        cout << nombre << " tene un peso saludable y su imc es de " << imc
             << endl;
      } else if (imc >= 25.0 || imc <= 29.9) {
        cout << nombre << " y tiene sobrepeso y su imc es de " << imc << endl;
      } else if (imc >= 30.0) {
        cout << nombre << " tiene obesidad y su imc es de " << imc << endl;
      }

      sumaedad = sumaedad + edad;
      sumaestatura = sumaestatura + estatura;
      sumapeso = sumapeso + masa;

      contjugadores++;

      cout << "¿quiere registrar otro jugador?" << endl;
      cin >> resp;

    } while (resp == "si");
    promedad = sumaedad / contjugadores;
    promestatura = sumaestatura / contjugadores;
    prompeso = sumapeso / contjugadores;
    cout << "los promedios de los jugadores es \n "
         << "promedio edad: " << promedad << endl
         << "promedio estratura: " << promestatura << endl
         << "promedio peso: " << prompeso << endl;

    cout<<"Quieres ingresar un nuevo equipo "<<endl;
    cin>> opc ;
  } while (opc == 1);

  return 0;
}
